packages/interactive_media_ads/pubspec.yaml

```diff
@@ -3,6 +3,7 @@ description: A Flutter plugin for using the Interactive Media Ads SDKs on Androi
 repository: https://github.com/flutter/packages/tree/main/packages/interactive_media_ads
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+interactive_media_ads%22
 version: 0.0.1
+publish_to: none
```

Comment:
Is it necessary to set this to `none`?

Reply:
I thought you had intended to set it to `none` after the initial publish, until it was actually ready. If not, we can remove this and do incremental releases for everything.

Reply:
Since it is pre-1.0, I think it's fine to have incremental releases. We also plan to release the Android implementation early for feedback.